data_diff/__main__.py

```diff
@@ -44,7 +44,12 @@
 @click.option("-v", "--verbose", is_flag=True, help="Print extra info")
 @click.option("-i", "--interactive", is_flag=True, help="Confirm queries, implies --debug")
 @click.option(
-    "-j", "--threads", default=None, help="Number of threads to use. 1 means no threading. Auto if not specified."
+    "-j",
+    "--threads",
+    default="1",
+    help="Number of worker threads to use per database. Default=1. "
+    "'auto' will use the maximum amount recommended by Python. "
```

Comment:
What does "recommended by Python" mean? 👂🏻

Reply:
It usually means equal to the number of cores on your local computer.

Reply:
(I know it's not necessarily the best option for database workers, but at least it keeps it within a reasonable range)

Reply:
Yeah it's a bit weird... I think we should just remove `auto` then 🤷🏻 I think it's more confusing than valuable.

Reply:
Fair enough